view/frontend/web/js/view/payment/method-renderer/omise-offsite-installment-method.js

```diff
@@ -0,0 +1,178 @@
+define(
+    [
+        'jquery',
+        'ko',
+        'mage/storage',
+        'Magento_Checkout/js/view/payment/default',
+        'Magento_Checkout/js/model/full-screen-loader',
+        'Magento_Checkout/js/action/redirect-on-success',
+        'Magento_Checkout/js/model/quote',
+        'Magento_Checkout/js/model/url-builder',
+        'Magento_Checkout/js/model/error-processor'
+    ],
+    function (
+        $,
+        ko,
+        storage,
+        Component,
+        fullScreenLoader,
+        redirectOnSuccessAction,
+        quote,
+        urlBuilder,
+        errorProcessor
+    ) {
+        'use strict';
+
+        return Component.extend({
+            defaults: {
+                template: 'Omise_Payment/payment/offsite-installment-form'
+            },
+
+            isPlaceOrderActionAllowed: ko.observable(quote.billingAddress() != null),
+
+            /**
+             * Get payment method code
+             *
+             * @return {string}
+             */
+            getCode: function () {
+                return 'omise_offsite_installment';
+            },
+
+            /**
+             * Initiate observable fields
+             *
+             * @return this
+             */
+            initObservable: function () {
+                this._super()
+                    .observe([
+                        'omiseOffsite',
+                        'installmentTerms'
+                    ]);
+
+                return this;
+            },
+
+            /**
+             * Get a checkout form data
+             *
+             * @return {Object}
+             */
+            getData: function () {
+                return {
+                    'method': this.item.method,
+                    'additional_data': {
+                        'offsite': this.omiseOffsite(),
+                        'terms': this.installmentTerms()
+                    }
+                };
+            },
+
+            /**
+             * Is method available to display
+             *
+             * @return {boolean}
+             */
+            isActive: function () {
+                return this.getOrderCurrency().toLowerCase() === 'thb' && this.getStoreCurrency().toLowerCase() === 'thb';
```

Comment:
Seems very wrong to be hardcoding this given that the `Capabilities` API gives us the currencies that are applicable for each backend, but I guess it works for now and helps to get Instalment payments `out of the door` faster

Reply:
this is hardcoded in exactly the same way for other payment methods, but good idea... will change it in future PR.